test/test_video_gpu_decoder.py

```diff
@@ -79,8 +82,9 @@ def test_seek_reading(self, keyframes, full_path, duration):
         ],
     )
     def test_metadata(self, video_file):
+        torchvision.set_video_backend("cuda")
```

Comment:
can we run this just once for all tests in this file since they all supposed to run with the cud backend?

Reply:
Good question. Let me try out :)

Reply:
For some reason that failed (putting it in a class header) so I'll keep it as is, and then re-do the tests later.